### PR TITLE
feat: Add bandit for security audit

### LINKED ISSUE #115: Add Bandit library
## Description

Add [Bandit](https://bandit.readthedocs.io/en/latest/) or similar.

> Bandit is a tool designed to find common security issues in Python code. To do this, Bandit processes each file, builds an AST from it, and runs appropriate plugins against the AST nodes. Once Bandit has finished scanning all the files, it generates a report.

### PR DESCRIPTION
**Pull request by vinigfer**
Installed bandit using `poetry add bandit`, which updated `poetry.lock`
and `pyproject.toml`. Then added a new call to make file to run it with
`make bandit`. For now running only on the main code, as running it on
the test folders showed a lot of possible security issues.

closes #115

### CODE REVIEW

**Comment by vinigfer** [comment]
Noticed that poetry automatically changed several places from 
`category = "dev"`
to
`category = "main"`
Can someone checked if that is OK? Never worked with poetry before

**Comment by supakeen** [comment]
The ``category``'s got changed because ``bandit`` was added as a ``main`` dependency (not a ``dev``) so its dependencies also got changed to ``main``.

**Comment by vinigfer** [comment]
OK! So pretty much all `main` occurrences on this PR needs to be fliped to `dev` right? Will fix that.

**Comment by vinigfer** [comment]
Done

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
```
Comment (camilamaia) [inline CHANGELOG.md:9]:
```suggestion

```
We keep all the changes under `Unreleased` section and only when we want in fact to create a new release, we move them to the proper release title, like this https://github.com/scanapi/scanapi/pull/212/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR10

So for now, we can leave everything under ##[Unreleased]

File: pyproject.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "scanapi"
-version = "1.0.5"
+version = "1.0.7"
```
Comment (camilamaia) [inline pyproject.toml:3]:
```suggestion
version = "1.0.5"
```
Since we will not bumpo the version now, we can leave this at is.

**Pull request merged**